src/Input/Input.tsx

```diff
@@ -113,7 +113,7 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(
 
     const icRightHidden =
       !currentIcRight || (currentIcRight === 'check' && !showValidationCheck)
-    const inputPaddingRight = icRightHidden ? 2 : 10
+    const inputPaddingRight = icRightHidden ? 2 : 12
```

Comment:
is it safe to modify it here for all consumers of this input ?

Reply:
I think so

Reply:
Maybe check for every input type? Modifying the base component seems very unsafe

Reply:
From what I saw, every input still renders as expected even with this modification.

Reply:
9227145d1429dd00450d17d8b7abfaec3edf5c51